product_code/lagom/hl_course_service/impl/src/main/resources/application.conf

```diff
@@ -0,0 +1,34 @@
+#
+#
+play.application.loader = de.upb.cs.uc4.hl.course.impl.HlCourseLoader
+
+jdbc-defaults.slick.profile = "slick.jdbc.PostgresProfile$"
```

Comment:
HlCourseService does not use a database, the database profile is not needed

Reply:
solved in new commit